### ISSUE TITLE
Tower Breakers typos

### ISSUE THREAD

**Issue by kapilace6** [TanishAggarwal5/coding_prep#4]
Typo `function`
https://github.com/TanishAggarwal5/coding_prep/blob/b3d7edddd6373b01b57be875efefe84229d96b8d/interview_prep/Tower%20Breakers/code.cpp#L18

I don't understand the comment, can you rephrase?
https://github.com/TanishAggarwal5/coding_prep/blob/b3d7edddd6373b01b57be875efefe84229d96b8d/interview_prep/Tower%20Breakers/code.cpp#L20

Add space between after `,`
https://github.com/TanishAggarwal5/coding_prep/blob/b3d7edddd6373b01b57be875efefe84229d96b8d/interview_prep/Tower%20Breakers/code.cpp#L29

**Comment by TanishAggarwal5** [comment]
fixed issues
